src/Microsoft.Health.Fhir.TemplateManagement/Client/ACRClient.cs

```diff
@@ -6,22 +6,30 @@
 using System;
 using System.IO;
 using System.Net;
+using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 using EnsureThat;
 using Microsoft.Azure.ContainerRegistry;
 using Microsoft.Azure.ContainerRegistry.Models;
+using Microsoft.Extensions.Logging;
 using Microsoft.Health.Fhir.TemplateManagement.Exceptions;
 using Microsoft.Health.Fhir.TemplateManagement.Models;
 using Microsoft.Health.Fhir.TemplateManagement.Utilities;
 using Microsoft.Rest;
 using Microsoft.Rest.Azure;
+using Polly;
 
 namespace Microsoft.Health.Fhir.TemplateManagement.Client
 {
     public class ACRClient : IOCIArtifactClient
     {
         private readonly IAzureContainerRegistryClient _client;
+        private static readonly ILogger _logger = TemplateManagementLogging.CreateLogger<ACRClient>();
+        private readonly AsyncPolicy _retryPolicy = Policy.Handle<HttpRequestException>().WaitAndRetryAsync(1, retryNumber => TimeSpan.FromMilliseconds(200), onRetry: (exception, retryCount, context) =>
```

Comment:
retry on all HttpRequestException?

Reply:
is there any other errors need to retry?

Reply:
According to the bug description, we need to fix this exception for now?